defaults/preferences/rwh-defaults.js

```diff
@@ -15,6 +15,7 @@ pref('extensions.replywithheader.enable', true);
 pref('extensions.replywithheader.header.from.style', 1);
 pref('extensions.replywithheader.header.tocc.style', 1);
 pref('extensions.replywithheader.header.lblseq.style', 1);
+pref('extensions.replywithheader.header.lang', 'en');
```

Comment:
Please change from `extensions.replywithheader.header.lang` to `extensions.replywithheader.header.locale`